ext/dom/tests/DOMNode_contains.phpt

```diff
@@ -0,0 +1,106 @@
+--TEST--
+DOMNode::contains()
+--EXTENSIONS--
+dom
+--FILE--
+<?php
+
+$dom = new DOMDocument();
+$dom->loadXML(<<<XML
+<!DOCTYPE HTML>
+<html xmlns:test="some:ns">
+    <head>
+        <title>my title</title>
+    </head>
+    <body>
+        <main>
+            <p>Hello, world!</p>
+            <p>Second paragraph</p>
+            <div><p>container</p></div>
+            <!-- comment -->
```

Comment:
Shouldn't this NodeType also be covered? Unless I misread, this node is not used.

Reply:
Forgot this yes, thanks for noticing.